src/commandsAndMenu.tsx

```diff
@@ -383,18 +384,24 @@ export function addCommands(
 
   /** Add git pull command */
   commands.addCommand(CommandIDs.gitPull, {
-    label: trans.__('Pull from Remote'),
-    caption: trans.__('Pull latest code from remote repository'),
+    label: args =>
+      args.discard
+        ? trans.__('Pull from Remote (Discard all)')
```

Comment:
I would use a wording mirroring the push case.

```suggestion
        ? trans.__('Pull from Remote (Force)')
```